pymc/func_utils.py

```diff
@@ -50,24 +50,26 @@ def find_constrained_prior(
         Lower bound to get `mass` % of probability of `pm_dist`.
```

Comment:
In distribution parameter, use only "Distribution" to use the alias and have it rendered as a link in the docs. Alias source: https://github.com/pymc-devs/pymc/blob/main/docs/source/conf.py#L85

Reply:
It is some lines above but I can't comment there